### PR TITLE
New stack opcodes

### LINKED ISSUE #237: Redesign stack opcodes
```c#
public enum OpCode : byte
{
    //State
    DEPTH = 0x43,
    //Reserved 0x44

    //DROPs
    DROP = 0x45,
    NIP = 0x46,
    //Reserved 0x47
    XDROP = 0x48,
    CLEAR = 0x49,

    //DUPs
    DUP = 0x4A,
    OVER = 0x4B,
    //Reserved 0x4C
    PICK = 0x4D,
    TUCK = 0x4E,
    //Reserved 0x4F

    //SWAPs
    SWAP = 0x50,
    ROT = 0x51,
    ROLL = 0x52,
    REVERSE3 = 0x53,
    REVERSE4 = 0x54,
    REVERSEN = 0x55,
}
```

### PR DESCRIPTION
**Pull request by erikzhang**
Closes #237

### CODE REVIEW

**Comment by erikzhang** [comment]
Ready for review.

File: src/neo-vm/ExecutionEngine.cs [hunk 1]
```diff
@@ -378,45 +366,16 @@ private bool ExecuteInstruction()
                         if (!context.EvaluationStack.TryRemove(n, out StackItem _)) return false;
                         break;
                     }
-                case OpCode.XSWAP:
+                case OpCode.CLEAR:
```
Comment (shargon) [inline src/neo-vm/ExecutionEngine.cs]: Does we need another CLEAR for alt stack?

File: src/neo-vm/ExecutionEngine.cs [hunk 2]
```diff
@@ -342,32 +343,19 @@ private bool ExecuteInstruction()
...
-                        Push(context.AltStack.Peek(-1));
-                        break;
-                    }
-                case OpCode.DUPFROMALTSTACK:
-                    {
-                        Push(context.AltStack.Peek());
-                        break;
-                    }
-                case OpCode.TOALTSTACK:
+                case OpCode.DEPTH:
```
Comment (shargon) [inline src/neo-vm/ExecutionEngine.cs]: Does we need another `DEPTH` for alt stack?

File: src/neo-vm/ExecutionEngine.cs (hunk 1 above)
Comment (erikzhang) [reply src/neo-vm/ExecutionEngine.cs]: AltStack is never needed and will be removed when slot opcodes are applied.

**Pull request merged**